digits/model/tasks/caffe_train.py

```diff
@@ -84,7 +84,7 @@ def __init__(self, **kwargs):
         self.solver = None
 
         self.solver_file = CAFFE_SOLVER_FILE
-        self.original_file = CAFFE_ORIGINAL_FILE
+        self.model_file = CAFFE_ORIGINAL_FILE
```

Comment:
Oh nevermind. We do the same for Torch. Rats.

However, you will want to be more careful with the upgrade path here.

* Bump the pickle version
* In `__setstate__`, upgrade from `original_file` to `model_file` appropriately